packages/vira/src/Vira/State/Type.hs

```diff
@@ -12,13 +12,6 @@ import Vira.Lib.Attic
 import Vira.Lib.Git (BranchName, CommitID)
 import Web.FormUrlEncoded (FromForm (fromForm), parseUnique)
 
-newtype RepoSettings = RepoSettings
-  { dummy :: Maybe Text
-  -- ^ Placeholder for future per-repo settings)
-  }
-  deriving stock (Generic, Show, Typeable, Data, Eq, Ord)
-  deriving anyclass (FromForm)
-
```

Comment:
`Vira.State.Types` holds all these high-level acid-state types. Is there a reason for moving this particuar type out of it?

Reply:
I thought `RepoSettings` isn’t part of acid-state anymore, esp. since it is intended to be controlled by downstream repos in the future.